addons/mail/models/mail_alias.py

```diff
@@ -93,6 +101,11 @@ def _alias_is_ascii(self):
                     alias.alias_name,
                 ))
 
+    @api.depends('alias_contact', 'alias_defaults', 'alias_model_id')
+    def _compute_alias_status(self):
+        """Reset alias_status to "not_tested" when fields, that can be the source of an error, are modified"""
```

Comment:
picky std (I use [pydocstyle](https://pypi.org/project/pydocstyle/) to be sure my docstring follow the PEP)

```suggestion
        """Reset alias_status to "not_tested" when fields, that can be the source of an error, are modified."""
```

Reply:
Thanks for the tip. It is changed.